README.md

```diff
@@ -42,26 +42,26 @@ On OS X, open Terminal and run the following command:
 defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false
 ```
 
-#### How can I bind `jj` to `<escape>`?
+#### How can I bind `jj` to `<Esc>`?
 
 1. Add the following to `settings.json` (open the Command Pallete and search for "User Settings"):
 
    ```
       "vim.insertModeKeyBindings": [
            {
                "before": ["j", "j"],
-               "after": ["<escape>"]
+               "after": ["<Esc>"]
```

Comment:
Are you referring to the capitalization? [VimDocs](http://vimdoc.sourceforge.net/htmldoc/intro.html#<Esc>) have it as a capital `E` so following suit.